addons/new-osio-addon/src/test/java/io/fabric8/launcher/osio/hoverfly/HoverflySimulationEnvironment.java

```diff
@@ -0,0 +1,33 @@
+package io.fabric8.launcher.osio.hoverfly;
+
+import static io.fabric8.launcher.osio.hoverfly.HoverflyRuleConfigurer.isHoverflyInSimulationMode;
+
+import org.junit.contrib.java.lang.system.ProvideSystemProperty;
+
+/**
+ * Gives a way to have fixed environment only when Hoverfly is in simulation mode.
+ */
+public class HoverflySimulationEnvironment extends ProvideSystemProperty {
```

Comment:
Two classes with the same name doing the same? We need to refactor that out to a shared module

Reply:
I do agree, it needs to be a library module for tests, where is the good place?

Reply:
Maybe we should approve this PR and create an issue for the new library module for tests.

Reply:
#129 has been created, you can approve/merge this one